Icon folder cleaning wave one

YesterdaysPromise opened the pull request:
## About The Pull Request

Due to a mental breakdown caused by unfathomable abomination that is icons folder, I swore to myself to one day clean it. Today is kind of that day. Been at it for around 6, you gotta understand I need a rest. I tracked most changes  in descriptions of commits if you are looking for details.

## Why It's Good For The Game

Saner spriters make better sprites. And also, just helps keep track of things.

## Changelog

:cl:
image: added sprites for different variants of scrolls.
image: modified couple posters with ghost pixels.
/:cl:

YesterdaysPromise committed:
Dyespray

Destroy dyespray.dmi, moves it into cosmetic.dmi also renames its  dyekit.dm into dyespray.dm.

YesterdaysPromise committed:
merges abductor_posters into posters

also organizes the file, and cleans some weird cases of stray pixels in it while I am at it.

YesterdaysPromise committed:
Effects vs Mob Effects

Moves etheral_crystal and genetics .dmi's into /mob/effects. Also merges obj/magic.dmi into effects/magic.dmi.

YesterdaysPromise committed:
Moves rune.dmi to obj/cult

YesterdaysPromise committed:
properly deletes obj/magic.dmi

YesterdaysPromise committed:
Magicks

dismantles wizard.dmi - moves shells into /mob/shells.dmi , artefacts into /obj/lavaland/artefacts.dmi ,effects into /effects/magic.dmi and gives scrolls their own .dmi also gives most scrolls their own sprites. Also moves heretic and cultist weapons into /obj/weapons/ with khopesh.dmi beign added. mostly, because wizards.dmi contained the mock-cultist khopesh

YesterdaysPromise committed:
boxcutter

merges boxcutter.dmi into tools.dmi

YesterdaysPromise committed:
moves stock_parts.dmi into /obj/assemblies/

YesterdaysPromise committed:
dismantles /obj/objects.dmi

most things go between /obj/debree.dmi , made from shards.dmi and added basic_machines.dmi into /obj/machines/

YesterdaysPromise committed:
combines device_syndicate.dmi and nuke_tools.dmi into syndicate_tools.dmi

ZephyrTFA commented:
Changelog is for player visible changes only

YesterdaysPromise commented:
> 

fair, fixed.

OrionTheFox commented on tgstation.dme:
```diff
@@ -5384,8 +5384,8 @@
 #include "interface\interface.dm"
 #include "interface\menu.dm"
-#include "interface\skin.dmf"
 #include "interface\stylesheet.dm"
+#include "interface\skin.dmf"
```
```suggestion
#include "interface\skin.dmf"
#include "interface\stylesheet.dm"
```
This keeps trying to change itself and the code doesnt like when its changed so we have to make sure it doesnt change itself 💀

YesterdaysPromise committed:
Update tgstation.dme

Co-authored-by: OrionTheFox <76465278+OrionTheFox@users.noreply.github.com>

YesterdaysPromise committed:
fix couple wrong allocations

YesterdaysPromise committed:
Merge branch 'icon-folder-cleaning' of https://github.com/YesterdaysPromise/tgstation into icon-folder-cleaning

YesterdaysPromise committed:
Bows aren't gun sybfolder anymore

ShizCalev commented:
*laughs in icon unit testing*

YesterdaysPromise committed:
dismantles stationobjs.dmi

can't sleep, ugh... well, anyway. splits stationobjs.dmi between wallmount.dmi , basic_machines.dmi and fluff.dmi mostly

YesterdaysPromise commented:
> _laughs maniacally in icon unit testing_

huh?

YesterdaysPromise committed:
creates a fluff folder in /obj/, stuffs mapper stuffs there

YesterdaysPromise committed:
moves contents of /mecha/ to mob, deletes the folder

YesterdaysPromise committed:
moves contents of /materials/ into turf, destroys the folder

YesterdaysPromise committed:
moves implants.dmi to /hud/

ChungusGamer666 commented:
heroic

YesterdaysPromise committed:
de-skill issues

YesterdaysPromise committed:
machinening

moves solars.dmi , robotics.dmi and smartfridge.dmi into /machines/ also moves all the machines from kitchen.dmi into /machines/kitchenmachines.dmi and all non-kitchen knives, along shivs from ddebree.dmi into /weapons/stabby

YesterdaysPromise committed:
whoop

YesterdaysPromise committed:
moves BSA from /lavaland/ to /machines/

YesterdaysPromise committed:
move money machine into /machines/

YesterdaysPromise committed:
dissmantles telescience.dmi

puts bluespace crystals and telecrystals into stack_objects.dmi , gps's into device.dmi, and remaining things into /machines/telepad.dmi

YesterdaysPromise commented:
alright seeing just how slow it checks, this will be the end of wave one

YesterdaysPromise committed:
naming changes

YesterdaysPromise committed:
Merge remote-tracking branch 'upstream/master' into icon-folder-cleaning

YesterdaysPromise committed:
fix

YesterdaysPromise committed:
uhuh

YesterdaysPromise committed:
fixes fixes

YesterdaysPromise committed:
all should be fine now?

YesterdaysPromise committed:
damn you tgstation.dme!

YesterdaysPromise committed:
resolves issues, take whichever it is

YesterdaysPromise committed:
typo moment

YesterdaysPromise committed:
uuuuh
